frontend/src/routes/(authenticated)/project/[project_code]/OrgList.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import t from '$lib/i18n';
+  import { Badge, BadgeList } from '$lib/components/Badges';
+  import type { Organization } from '$lib/gql/types';
+
+  export let organizations: Partial<Organization>[] = [];
```

Comment:
This type definition can be made more specific by using https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys to specify that the organization list passed in needs to have the `id` and `name` properties. That way if you try to pass it the result of a GraphQL query that only has `organizatoins { id }` in it, there will be a type error. As it's currently written, passing this the result of `organizations { id }` would not result in a type error, but would result in a blank-looking list with no org names visible.

The exact way to write the `Pick<...>` type is left as an exercise for the reader. :grin: You should be able to figure it out pretty easily from the example in the Typescript documentation. But if you need help, ask and I'll help you out with it.